modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,52 +1,100 @@
-= Release Notes for Couchbase Server 6.0 BETA
+= Release Notes for Couchbase Server 6.0
 
-Couchbase Server 6.0 Beta is an important release that adds the Analytics service to Couchbase Data Platform.
+Couchbase Server 6.0 is an important release that adds the Analytics service to the Couchbase Data Platform.
 It also includes enhancements to the Eventing and Search services.
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
-[IMPORTANT]
-====
-This Beta version is intended for development and testing purposes only.
-We consider Beta releases to have some rough edges and bugs, and may change significantly before the final GA release.
-This release is *not supported* in production.
+[#release-600]
+== Release 6.0
 
-The Beta release does *not support* upgrading from an earlier version of Couchbase Server.
-====
+Couchbase Server 6.0 was released in October 2018.
+
+[#changes-in-behavior-600]
+=== Major Behavior Changes
+
+There are a few significant changes to the production-ready version of the Analytics service.
+You'll find this information handy if you've used the Analytics service in any of the Developer Preview versions of Couchbase Server.
+
+* Bucket association is now automatic
++
+The Developer Preview versions required you to explicitly associate the Couchbase Server buckets and Analytics buckets.
+For example:
++
+----
+CREATE BUCKET ha WITH {"name":"health"}
+----
++
+The Analytics service buckets now automatically mirror Couchbase Server buckets. For instance, after using the previous example command, other Analytics DDL commands would be able to refer to the bucket `health` without any extra configuration.
```

Comment:
What does "the previous example command" refer to? 
In 6.0 there is no command required to be able to create a dataset on a bucket.

Reply:
The previous example command refers to the following statement:
CREATE BUCKET ha WITH {"name":"health"}

I've updated this to say: 
The Analytics service buckets now automatically mirror Couchbase Server buckets and no special configuration is required to be able to create a dataset on a bucket.